py-polars/polars/cfg.py

```diff
@@ -105,6 +106,19 @@ def set_global_string_cache(cls) -> type[Config]:
         toggle_string_cache(True)
         return cls
 
+    @classmethod
+    def set_with_columns_kwargs(
+        cls, enable: Optional[bool] = None
+    ) -> Union[type[Config], bool]:
+        """
+        Enable experimental support for kwargs in `with_columns` method.
+        """
+        if enable is None:
```

Comment:
I don't think we need an env var here. Maybe we can set a boolean flag on the class? 

This ensure that if multiple users on the same server use polars, this setting doesn't interfere. (I know others do, but as we don't have to cross the FFI boundary here, it is easier to set this in the python running instance).

Reply:
Done - definitely cleaner.